src/Dropdown.php

```diff
@@ -384,6 +384,9 @@ public static function show($itemtype, $options = [])
                     $paramskblinks,
                     false
                 );
+                if ($item->fields['knowbaseitemcategories_id'] != $params['value']) {
```

Comment:
Can you explain this ?

Reply:
$item is ITILCategory, but in the simplified interface the user has no rights to ITILCategory, so the condition in line 292 is false and $item remains empty.

```
if ($item->canView()) {
                if (
                    $params['value']
                    && $item->getFromDB($params['value'])
                    && $item->canViewItem()
                ) {
                    $options_tooltip['link'] = $item->getLinkURL();
                } else {
                    $options_tooltip['link'] = $item->getSearchURL();
                }
}
```

This is to fill it in if needed.

Reply:
But, why testing `$item->fields['knowbaseitemcategories_id']` ? You should probably use `if ($item->isNewItem())` and add a comment to explain this.